src/actions/walletConnectActions.js

```diff
@@ -105,6 +106,7 @@ export const connectToWalletConnectConnectorAction = (uri: string) => {
     }
 
     dispatch(logEventAction('walletconnect_connector_requested'));
+    isLogV2AppEvents() && dispatch(logEventAction('v2_wallet_connect_dapp_connect'));
```

Comment:
Is this line not supposed to go _inside_ the `connector.on` event listener below?